### ISSUE TITLE
Does this still work with current Twine and Tracery?

### ISSUE THREAD

**Issue by lulamorashi** [incobalt/Trice#3]
I couldn't find the vendor/seedrandom.js in the Tracery depository, does this mean that Trice will no longer work?

**Comment by incobalt** [comment]
I am uncertain what you mean. Vendor/seedrandom.js is in the Tracery repository (see: https://github.com/galaxykate/tracery/tree/tracery2/js/vendor ). I noticed that there was a newer Tracery repository (https://github.com/galaxykate/tracery2) but it also has vendor/seedrandom.js (https://github.com/galaxykate/tracery2/tree/master/js/vendor). I will note that I have not tested if Trice will work with this second repository, but the lack of mods-eng-basic.js suggests that it won't work without editing the story Javascript.

I also haven't much tested Trice against Twine 2.4+ or SugarCube 2.32+, so it may not work properly in newer versions of Twine and SugarCube. I did just load Trice Example up in Twine 2.6.2 and compiled it with SugarCube 2.36.1, and it ran fine. In doing these tests, I had a thought that perhaps you are trying to test your story using the Test or Play buttons in Twine, which, as discussed here: https://github.com/incobalt/Trice#setting-up-trice

I hope any of that helps!
